Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.py

```diff
@@ -2039,6 +2037,44 @@ def get_item_as_eml(client: EWSClient, item_id, target_mailbox=None):      # pra
     return None
 
 
+def handle_attached_email_with_incorrect_id(attached_email):
```

Comment:
missing type hint (sorry for having missed it earlier)

Reply:
👍